static/scss/sitewide4.scss

```diff
@@ -569,67 +563,6 @@ a#readon {
 	margin-left:5px;
 }
 
-/* Explore column */
-#js-leftcol {
-    float:left;
-    width:235px;
-    margin-bottom: 20px;
-
-    a {
-        font-weight:normal;
-
-        &:hover{
-            text-decoration:underline;
-        }
-    }
-
-    .jsmod-content {
-        border: solid 1px $pale-blue;
-        @include border-radius(0, 0, 10px, 10px);
-    }
-
-    ul {
-        &.level1 > li > a, &.level1 > li > span {
-            border-bottom:1px solid $pale-blue;
```

Comment:
the 1 pixel borders are nice on desktop at least - would it be hard to keep them?

Reply:
when in doubt, don't change things - they make returning users comfortable